### DISCUSSION TITLE
noob question on calling from HomeAssistant as a service

### DISCUSSION THREAD

**Discussion by tayacar** [rudybrian/PyNavienSmartControl#23] [Q&A] [answered]
Not sure if you can help, I'm struggling on how to just call the python code as a service within home assistant. I have moved the package to within a <config>/python_scripts/ and updated the credentials file. I can't seem to even get home assistant to see the python file within the folders. How do I call PyNavienSmartControl as a service to turn on/off my Navien tankless? Any directions would be helpful. Thanks in advance!

**Comment by rudybrian** [comment, marked as answer]
As-is PyNavienSmartControl is just an interface library and sample applications that a Python developer could use with their application and isn't written specifically for Home Assistant. You probably want to have a look at [nikshriv/hass_navien_water_heater](https://github.com/nikshriv/hass_navien_water_heater) that is based on PyNavienSmartControl since this likely does what you want.

**Reply by tayacar** [reply]
Thank you!!! I was able to get everything setup last night!! So appreciated.
